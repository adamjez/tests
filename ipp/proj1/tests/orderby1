#TEST22
#AUTOR: xjezad00

CommandLine =--input=./tests/test11.in --query='SELECT book FROM ROOT WHERE genre = "Computer" ORDER BY price DESC' --root=ROOT


ExpectedReturnValue = 0

# Check if is not stderr empty:  Off - 0 On - 1 
CheckStderr = 1

# Check if is stdout empty:  Off - 0 On - 1 
CheckStdout = 1

# Compare stdout with ref:  Off - 0 On - 1 	(ref. file: ./tests/[TestName].out)
CmpStdout = 1

# Check if is not stdout empty: Off - 0 On - 1
CheckStdoutNot = 0

