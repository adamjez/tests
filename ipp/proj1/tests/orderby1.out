<?xml version="1.0" encoding="utf-8"?>
<ROOT>
<book id="bk112" order="1">
      <author>Galos, Mike</author>
      <title>Visual Studio 7: A Comprehensive Guide</title>
      <genre>Computer</genre>
      <price>49.95</price>
      <publish_date>2001-04-16</publish_date>
      <description>Microsoft Visual Studio 7 is explored in depth,
      looking at how Visual Basic, Visual C++, C#, and ASP+ are 
      integrated into a comprehensive development 
      environment.</description>
   </book>
<book id="bk101" order="2">
      <author>Gambardella, Matthew</author>
      <title>XML Developer's Guide</title>
      <genre>Computer</genre>
      <price>44.95</price>
      <publish_date>2000-10-01</publish_date>
      <description>An in-depth look at creating applications 
      with XML.</description>
   </book>
<book id="bk110" order="3">
      <author>O'Brien, Tim</author>
      <title>Microsoft .NET: The Programming Bible</title>
      <genre>Computer</genre>
      <price>36.95</price>
      <publish_date>2000-12-09</publish_date>
      <description>Microsoft's .NET initiative is explored in 
      detail in this deep programmer's reference.</description>
   </book>
<book id="bk111" order="4">
      <author>O'Brien, Tim</author>
      <title>MSXML3: A Comprehensive Guide</title>
      <genre>Computer</genre>
      <price>36.95</price>
      <publish_date>2000-12-01</publish_date>
      <description>The Microsoft MSXML3 parser is covered in 
      detail, with attention to XML DOM interfaces, XSLT processing, 
      SAX and more.</description>
   </book>
</ROOT>